src/cross_section/src/cross_section.cpp

```diff
@@ -560,6 +612,42 @@ CrossSection CrossSection::Offset(double delta, JoinType jointype,
   return CrossSection(ps);
 }
 
+CrossSection CrossSection::Hull(
+    const std::vector<CrossSection>& crossSections) {
```

Comment:
We do have `Decompose` for `CrossSection`, and `Compose` is just an alias for a batch union (to make sure that the output is still a valid `CrossSection`). Allowing a set of `CrossSections` to be hulled like here is nice since then a needless clipping operation is avoided.

Reply:
I guess, but they can already make a vector of polygons pretty easily. I'd like to keep our API small where possible.

Reply:
Nice job on `Decompose` by the way - I'd totally forgotten you already put that in. 💯

Reply:
> I guess, but they can already make a vector of polygons pretty easily. I'd like to keep our API small where possible.

I think this ones worth having. A lot of basic use case for 2d hull is making rounding thing to extrude, and being able to do that directly with some Circles is pretty nice.